src/java/org/apache/cassandra/db/ConsistencyLevel.java

```diff
@@ -264,4 +264,9 @@ private void requireNetworkTopologyStrategy(AbstractReplicationStrategy replicat
             throw new InvalidRequestException(String.format("consistency level %s not compatible with replication strategy (%s)",
                                                             this, replicationStrategy.getClass().getName()));
     }
+
+    public static ConsistencyLevel fromStringIgnoreCase(String s)
```

Comment:
why?  Is this now trying to make even more things valid?  Is this needed for this patch?